pkgs/applications/video/plex-mpv-shim/default.nix

```diff
@@ -1,15 +1,13 @@
-{ lib, buildPythonApplication, fetchFromGitHub, mpv, requests, python-mpv-jsonipc, pystray, tkinter
+{ lib, buildPythonApplication, fetchPypi, mpv, requests, python-mpv-jsonipc, pystray, tkinter
 , wrapGAppsHook, gobject-introspection }:
 
 buildPythonApplication rec {
   pname = "plex-mpv-shim";
   version = "1.11.0";
 
-  src = fetchFromGitHub {
-    owner = "iwalton3";
-    repo = pname;
-    rev = "refs/tags/v${version}";
-    sha256 = "sha256-hUGKOJEDZMK5uhHoevFt1ay6QQEcoN4F8cPxln5uMRo=";
+  src = fetchPypi {
```

Comment:
In general we should prefer using the github source, can we build the shaders ourselves?

Reply:
We can -- it would need to be a separate derivation (or even separate package) as it comes from a different repository (https://github.com/iwalton3/default-shader-pack). I actually started with that, but found that [jellyfin-mpv-shim](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/video/jellyfin-mpv-shim/default.nix) (which uses the same shaders) used `fetchPypi` instead, which includes the shaders as part of a single source, and chose to follow that pattern instead for consistency. Can certainly switch back to Github source and add shaders separately here -- not an issue.

Reply:
Switched back to github source and added shader package.